src/components/UsersList/UsersList.tsx

```diff
@@ -50,37 +70,57 @@ export function UsersList({
     }
   }
 
-  function removeUserCallback(user: UserObj) {
-    setRemovedUserGuids([...removedUserGuids, user.guid]);
-    openModal(user);
+  // Sorting
+  interface SortParamsTable {
+    [colName: string]: string;
   }
+  const sortParamsTable = {
+    'account name': 'username',
+    'organization roles': 'orgRolesCount',
+    'access permissions': 'spaceRolesCount',
+    expires: 'daysToExpiration',
+    'last login': 'lastLogonTime',
+  } as SortParamsTable;
 
-  const [removedUsername, setRemovedUsername] = useState('');
+  function usersSorted(usersList: Array<UserOrgPage>): Array<UserOrgPage> {
+    return sortObjectsByParam(usersList, sortParam, sortDir);
+  }
+
+  const onSortAction = (colName: string) => {
+    if (sortParamsTable[colName] === sortParam) {
+      if (sortDir === 'asc') setSortDir('desc' as SortDirection);
+      if (sortDir === 'desc') setSortDir('asc' as SortDirection);
+    } else {
+      setSortDir('asc' as SortDirection);
+      setSortParam(sortParamsTable[colName]);
+    }
+  };
+
+  // Modal
+  const modalHeadingId = (name: string) => `removeUserSuccess-${name}`;
 
   function closeModal(): undefined {
     setRemovedUsername('');
   }
-  function openModal(user: UserObj): undefined {
+
+  function openModal(user: UserOrgPage): undefined {
     setRemovedUsername(user.username);
   }
 
-  const onSearchAction = (value: string) => {
-    if (value.trim().length <= 0) {
-      setSearchValue('');
-    } else {
-      setSearchValue(value.trim());
-    }
-  };
+  function removeUserCallback(user: UserOrgPage) {
+    setRemovedUserGuids([...removedUserGuids, user.guid]);
+    openModal(user);
+  }
 
-  const modalHeadingId = (name: string) => `removeUserSuccess-${name}`;
+  // Helpers
   const currentUsers = usersSorted(usersFiltered(users));
   const usersResultsText = currentUsers.length === 1 ? 'user' : 'users';
 
   return (
     <>
       <ListSearchInput
         onSubmit={onSearchAction}
-        labelText="search the list of users by username"
+        labelText="Find account names that match:"
```

Comment:
In the rendered UI, I’m seeing “Find account names that match:” twice inside the form: once as a bolded paragraph before the search form, and then again in a visually-hidden label inside the search form. Would it be possible to just show the label, and drop the paragraph?

Reply:
I was trying to work within the bounds of the `usa-search` styles here and got carried away. You're right, there should just be one label. I'll drop the p tag, move the label to be above the form element, and make the label visible. I think as long as the label's `for` and the input's `id` match up, a label doesn't _have_ to be in a form element—is that correct?

Reply:
> I think as long as the label's for and the input's id match up, a label doesn't have to be in a form element—is that correct?

Yeah, spot on! Totally fine to have a `label` sitting outside a form.

(Out of curiosity, is there a reason we can’t style the label that’s inside the form?) (Just asking for my edification here: if the answer’s too lengthy for PR chatter, feel free to disregard.)

Reply:
In short, `.usa-search` is flexed as a row, and I didn't want to override those selectors.

Reply:
Thanks, Eleni!

And the new label works great 🎉